core/frontend/helpers/tpl/content-cta.hbs

```diff
@@ -7,6 +7,9 @@
         {{#has visibility="members"}}
             <h2>This post is for subscribers only</h2>
         {{/has}}
+        {{#if customVisibilityText}}
```

Comment:
If we're introducing this property as part of the theme API, has it passed design review?

Reply:
@ErisDS Its pending design review and don't expect the property to go out in current form for theme API. This PR was purely for demo purposes, sorry should have added a note in description clarifying(done now).